src/mixins/crownConfig.js

```diff
@@ -188,6 +191,12 @@ export default {
       });
     },
     configureCrown() {
+      if (this.crownConfigured) {
+        return;
+      }
+
+      this.crownConfigured = true;
+
```

Comment:
This seems a little weird. What do these lines achieve and why are they needed?

Reply:
It's to prevent this function from running twice. Running it twice duplicates the crown buttons. Without highlight on drop, this method was only called once when the element was clicked. But with the crown already visible on drop, I don't want this function called again if a user clicks the element.

I'll see if I can clean this up.

Reply:
I removed `crownConfigured` and made a single call to `configureCrown` within `setUpCrownConfig`. This may have some performance implications (as the call to `configureCrown` was deferred to help with performance on initial load), but as long as tests continue to pass, performance issues will just have to be re-assessed in the future; I think this is a fair trade-off.